config/initializers/_constants.rb

```diff
@@ -1,5 +1,8 @@
 # frozen_string_literal: true
 
+# Path to login via api key
+AUTH_API_PATH = '/auth/api'
+
```

Comment:
```suggestion
```

I think it's an artifact of the previous version that uses full connection and not Authorization header